modules/pam_access/pam_access.c

```diff
@@ -763,19 +768,47 @@ network_netmask_match (pam_handle_t *pamh,
 	    netmask_ptr = number_to_netmask(netmask, addr_type,
 		netmask_string, MAXHOSTNAMELEN);
 	  }
-	}
+
+	/*
+	 * Although isipaddr() has already converted the IP address,
+	 * we call getaddrinfo here to properly construct an addrinfo list
+	 */
+	memset (&hint, '\0', sizeof (hint));
+	hint.ai_flags = 0;
+	hint.ai_family = AF_UNSPEC;
```

Comment:
`ai_family` should probably be assigned to `addr_type` instead of `AF_UNSPEC`.

Reply:
The whole "hint" isn't necessary at all here, removed.